src/mappers/country/country_mapper.h

```diff
@@ -16,9 +16,10 @@ class CountryMapper
 {
   public:
    CountryMapper() = default;
-   explicit CountryMapper(std::map<int, std::string> country_mappings): country_mappings_(std::move(country_mappings))
+   explicit CountryMapper(std::map<int, std::string>&& country_mappings): country_mappings_(std::move(country_mappings))
    {
    }
+   explicit CountryMapper(const std::map<int, std::string>& country_mappings): country_mappings_(country_mappings) {}
```

Comment:
Unless I'm very mistaken, this won't have any savings over the original. See https://abseil.io/tips/117 and https://mpark.github.io/programming/2014/06/07/beware-of-perfect-forwarding-constructors/

Reply:
it will eliminate a copy/move if country_mappings is a nonconst rvalue, such as when we're creating this from the builder.

IIUC, the main gotcha with moving is that you can't move `const` objects, which makes sense as they could theoretically be in read-only memory.

Reply:
Also there's template weirdness with perfect forwarding n stuff that I don't quite understand, but we're not using a template for this class so we can avoid that

Reply:
Guaranteed copy elision should be handling moves from temporaries for us. And yeah, perfect forwarding is cool, but less commonly useful for us.

Reply:
I'll change this around to rely on copy elision/NRVO when possible.